.storybook/environment.js

```diff
@@ -0,0 +1,34 @@
+const {
+  TwingEnvironment,
+  TwingFunction,
+  TwingFilter,
+  TwingLoaderRelativeFilesystem,
+} = require('twing');
+const twigBEM = require('bem-twig-extension');
+const twigDrupal = require('twig-drupal-filters');
+const twigAddAttributes = require('add-attributes-twig-extension');
+
+const loader = new TwingLoaderRelativeFilesystem();
+const environment = new TwingEnvironment(loader, {
+  autoescape: false,
+  debug: false,
+});
+
+const asPromise = (callback) => (...args) => {
+  return Promise.resolve(callback(...args));
+};
+
+const TwigToTwing = {
```

Comment:
As you can see here, we have a `TwigToTwing` object. I tried to put together a quick "glue" object that would allow us to use twig.js extensions with Twing, but I think that's pretty hacky, and it's not ideal. It does _kind of_ work, however Twing expects different return values from filters/fns than twig.js. So, we need to re-implement `bem-twig-extension`, `twig-drupal-filters`, and `add-attributes-twig-extension` so that they're compatible with Twing.

Reply:
Additionally, the `twig-testing-library` doesn't look like it supports Twing, so we'll need to fork/modify it, or post a PR that makes it work with both systems.

Reply:
Take a look at [this Twing loader](https://github.com/wingsuit-designsystem/wingsuit/blob/bcb61e38051f7738ad94195248f74280b741b8b5/packages/pattern/src/TwingRenderer.ts), it  might point us in the right direction.